models/token.go

```diff
@@ -21,11 +21,11 @@ type AccessToken struct {
 	Sha1 string `xorm:"UNIQUE VARCHAR(40)"`
 
 	Created           time.Time `xorm:"-"`
-	CreatedUnix       int64
+	CreatedUnix       int64     `xorm:"INDEX"`
```

Comment:
@ethantkoenig A clustered index will actually rearrange the physical rows within the table. A non-clustered index will not affect the physical storage. ORDER BY clauses still work the same. Clustered indexes are useful where you often select a large number of rows from the index. The query optimiser will the perform a table-scan as it's more efficient.